autotest/pymod/test_py_scripts.py

```diff
@@ -67,8 +68,19 @@ def get_py_script(script_name):
 # Runs a Python script
 # Alias of run_py_script_as_external_script()
 #
-def run_py_script(script_path, script_name, concatenated_argv):
-    return run_py_script_as_external_script(script_path, script_name, concatenated_argv)
+def run_py_script(script_path: str, script_name: str, concatenated_argv: str,
+                  run_as_script: bool = True, run_as_module: bool = False):
+    result = None
+    if run_as_module:
+        try:
+            module = importlib.import_module('osgeo_utils.' + script_name)
+        except ImportError:
+            module = importlib.import_module('osgeo_utils.samples.' + script_name)
+        argv = [module.__file__] + concatenated_argv.split(' ')
```

Comment:
might be fragile if some arguments include a space and are quoted

Reply:
Thanks, I've replaced that split with `shlex.split`.